pimpmycause/pimpmycause/templates/profiles/profile.html

```diff
@@ -6,39 +6,93 @@
+
+<!--                                           <label for="image"> Upload profile picture </label>
+ -->                                           <input class="button" type="submit" value="Upload Picture">
+                                            </div>
+                                        {% else %}
+                                            <img src="{{profile_update_form.image.value}}" alt="Profile Image">
+                                            <input class="button" type="submit" value="Change Picture">
+
+                                        {% endif %}
+<!-- 
+                                        <label for="image"> Change profile picture </label>
+                                        {{ profile_image_upload_form.media }}
+                                        {{ profile_image_upload_form.as_p }} -->
                                     </div>
-                                    <div class="small-12 columns">
-                                        {{ profile_update_form.website.errors }}
-                                        <input type="url" name="website" placeholder="www.example.com" value='{{profile_update_form.website.value|default_if_none:""}}' required>
+                                    <div class="small-6 large-6 medium-6 columns">
+                                        {{ profile_update_form.name.errors }}
+                                        <label for="name"> Name </label>
+                                        <input type="text" name="name" placeholder="Name" value='{{ profile_update_form.name.value|default_if_none:"" }}' required>
+
+                                        {{ profile_update_form.surname.errors }}
+                                        <label for="surname"> Surname </label>
+                                        <input type="text" name="surname" placeholder="Surname" value='{{ profile_update_form.surname.value|default_if_none:"" }}' required>
                                     </div>
                                 </div>
-                                <div class="row">
-                                    <div class="medium-6 large-4 small-6 columns">
-                                        <input class="button" type="submit" value="Update">
-                                    </div>                                
+                            </div>
+                            <div class="row">
+                                <h2> Social</h2>
+                            </div>
+                            <div class="row">
+                                <div class="large-6 medium-6 small-12 columns">
+                                    {{ profile_update_form.linkedin.errors }}
+                                    <label for="linkedin"> LinkedIn </label>
+                                    <input type="url" name="linkedin" placeholder="LinkedIn" value='{{ profile_update_form.linkedin.value|default_if_none:"" }}' required>
```

Comment:
if the field is required do we need like a red '*' and an explanation saying it's required ? or something else that helps the user understand it's required ? :)